lib/rubocop/config_validator.rb

```diff
@@ -0,0 +1,238 @@
+# frozen_string_literal: true
+
+require 'pathname'
+
+module RuboCop
+  # ...
```

Comment:
I guess you meant to write some description here. ;-)

Reply:
Ouch! :grimacing: